erpnext/stock/doctype/delivery_note/delivery_note.py

```diff
@@ -223,6 +223,8 @@ def on_submit(self):
 		if self.issue_credit_note:
 			self.close_sales_orders()
 
+		if not self.delivered: self.status = "To Deliver"
```

Comment:
Couldn't this also be "Out for Delivery"?

Reply:
need discussion.

Reply:
This is correct. When the delivery Note is submitted it will always be "To Deliver" unless delivered is checked